src/test/groovy/LogoutIT.groovy

```diff
@@ -0,0 +1,28 @@
+import geb.spock.GebReportingSpec
+
+class LogoutIT extends GebReportingSpec {
+
+  private static String baseURI
+
+  void setupSpec() {
+    baseURI = System.getProperty('gretty.baseURI')
+  }
+
+  void setup() {
+    go "${baseURI}/test/set-up/"
+  }
+
+  def 'log out'() {
+    given:
+    go "${baseURI}"
+    $('form').username = 'user'
+    $('form').password = 'password'
+    $('form button[type=submit]').click()
+    when:
```

Comment:
What do you think about adding an empty line before each block:

```groovy
given:
go "${baseURI}"
$('form').username = 'user'
// ...

when:
$('#logout').click():
// ...

then:
$('#logout').text() == null;
// ...
```

Visual empty lines increases the readability. What is your opinion on this?